src/backend/input.rs

```diff
@@ -216,6 +231,9 @@ pub trait InputBackend: Sized {
     /// Get current `InputConfig`
     fn input_config(&mut self) -> &mut Self::InputConfig;
 
+    /// Called to inform the Input backend about a new focused Output for a `Seat`
+    fn set_output_metadata(&mut self, seat: &Seat, output: &Output);
```

Comment:
I expect to get wired behavior then like [some bugs](https://github.com/Cloudef/wlc/issues/181) in wlc, I did encounter, because of floating point rounding.

I don't know if any other information are necessary. I would just go with `fn set_output_metadata(&mut self, width: u32, height: u32)` then and just refactor it, if we ever need more information.